launcher-core/src/main/scala/bloop/launcher/bsp/BspBridge.scala

```diff
@@ -78,7 +78,7 @@ final class BspBridge(
         bspServerStatus = Some {
           bspCmd.mkString(" ") -> {
             // Make it verbose so that bsp preparation logs are emitted
```

Comment:
This comment seems no longer valid? :slightly_smiling_face:

Reply:
It's not, thanks! I removed it